jenkins/check_copyright.sh

```diff
@@ -18,7 +19,7 @@ rm -f $ERRFILE
 rm -f $ERRFILETEMP
 
 echo "Check Copyright > Begin Checking..."
-git diff --name-only `git merge-base origin/$ghprbTargetBranch HEAD` HEAD |
+git diff --name-only `git merge-base origin/master HEAD` HEAD |
```

Comment:
This instruction didn't work - edited to fix (by bringing in line with the logic in the other check scripts